README.md

```diff
@@ -13,6 +13,14 @@ $ GO111MODULE=on go mod vendor
 $ modvendor -copy="**/*.c **/*.h **/*.proto" -v
 ```
 
+If you have additional directories that you wish to copy which are not specified
+under `./vendor/modules.txt`, use the `-additional_dirs` flag, e.g.:
+
+```
+$ GO111MODULE=on go mod vendor
+$ modvendor -copy="**/*.c **/*.h **/*.proto" -v -additional_dirs="github.com/grpc-ecosystem/grpc-gateway:github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis/google/api,github.com/grpc-ecosystem/grpc-gateway:github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis/google/rpc,github.com/prometheus/client_model:github.com/prometheus/client_model"
```

Comment:
TBH, I don't understand the syntax of "additional_dirs". I'd also prefer the term, "extra" just, `-extra=dir1` `-extra=dir2` ... or ...... single -extra flag with comma separate dirs is fine, but why are there colons in there too..?

Reply:
or, `-include` .. ? or some other name .. should be short

Reply:
it needs to map `package name:package directory`.

e.g.

`github.com/grpc-ecosystem/grpc-gateway` needs to include `github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis/google/api`, hence `github.com/grpc-ecosystem/grpc-gateway:github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis/google/api`.

do you have a cleaner way of representing that?

Reply:
i've made it just comma separated by comparing prefixes against import path. works for me.